File: js/compleated/35.js
var s = function(nums, target) {
    nums.push(target)
    nums.sort(function(a, b) {
        return a - b
    })
    console.log(nums)
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] == target) {
            return i
        }
    }
};

let nums = [1,2,3,4,5,10]
let target = 2

console.log(s(nums, target));

File: rust/examples/35.rs

fn main() {
    let nums = vec![1,3,5,6];
    let target = 1;

    println!("{:?}", s(nums, target));
}


fn s(nums: Vec<i32>, target: i32) -> i32 {
    let mut n = nums.clone();
    if !n.contains(&target) {
        n.push(target);
        n.sort();
    }
    n.binary_search(&target).unwrap() as i32
}